packages/mds-policy/api.ts

```diff
@@ -146,7 +146,7 @@ function api(app: express.Express): express.Express {
           get_published: [get_published = null],
           get_unpublished: [get_unpublished = null]
         } = scopes.includes('policies:read')
-          ? parseRequest(req, { parser: x => (x ? JSON.parse(x) : null) }).query('get_published', 'get_unpublished')
+          ? parseRequest(req, { parser: { fn: JSON.parse } }).query('get_published', 'get_unpublished')
```

Comment:
If we don't expect to add a bunch more options, I would advocate that rather than:
```ts
{ parser: { fn: JSON.parse, type: 'single'  } }
```
We just go with:
```ts
{ parser: JSON.parse, type: 'single' }
```